pygpu/_elemwise.pyx

```diff
@@ -60,6 +60,8 @@ cdef class arg:
             self.a.flags |= GE_WRITE
         if scalar:
             self.a.flags |= GE_SCALAR
+	if self.a.flags == 0:
```

Comment:
What if it is not readable, not writable and scalar? It won't raise and error. Is that normal?